basic/src/test/java/org/jboss/hal/testsuite/test/configuration/undertow/UndertowOperations.java

```diff
@@ -194,9 +199,15 @@ public void removeBufferPool(String name) {
 
     public String createSocketBinding() {
         String name = "UndertowSocketBinding_" + RandomStringUtils.randomAlphanumeric(6);
-        ResourceAddress address = socketBindingAddressTemplate.resolve(context, name);
-        Map<String, String> params = ImmutableMap.of("port", String.valueOf(ThreadLocalRandom.current().nextInt(1000, 9999)));
-        executeAddAction(address, params);
+        try (OnlineManagementClient client = ManagementClientProvider.createOnlineManagementClient()) {
+            client.apply(new AddSocketBinding.Builder(name)
+                    .port(ThreadLocalRandom.current().nextInt(1000, 9999))
+                    .build());
+        } catch (CommandFailedException e) {
```

Comment:
I prefer rethrow instead of continuing to run the test. BTW you probably wanted to use `printStackTrace` instead of `getStackTrace`.